core/chains/evm/types/types.go

```diff
@@ -188,6 +188,22 @@ func (r *Receipt) Value() (driver.Value, error) {
 	return json.Marshal(r)
 }
 
+func (r Receipt) GetStatus() uint64 {
```

Comment:
Mixed receiver types?
```suggestion
func (r *Receipt) GetStatus() uint64 {
```

Reply:
...ah good catch - there was some pre-existing mixed receiver types already. I will also fix those